test/powershell/Modules/Microsoft.PowerShell.Utility/assets/TestImportCsv2.csv

```diff
@@ -1,5 +1,5 @@
-Column1,Column2,Column 3
-data1,1,A
-data2,2,B
-data3,3,C
-data4,4,D
+﻿Column1,Column2,Column 3
```

Comment:
I'd like to see another test case similar to this one that contains a comment as the first line. It can be as simple as "# Comment". This case is covered by the existing Import-Csv cmdlet and it'd be nice to prove it still works as well.